Documentation/Signoffs/Sensors.md

```diff
@@ -0,0 +1,145 @@
+# Sensors System 
+
+  
+
+## Subsystem Function  
+
+  
+
+The role of the sensor subsystem is to send signals—communicating the status of the device occupancy of the presently forefront compartment sections on all three dial layers—to the PLC to update certain tags, so that the PLC's ladder logic code can properly use that information ad hoc.   
+
+  
+
+ 
+
+## Constraints  
```

Comment:
These are still a bit high level. What does it take to be able to detect presence vs absence?